### ISSUE TITLE
I get this error when running node 4.4.5

### ISSUE THREAD

**Issue by kevbook** [AndreasMadsen/trace#29]
I get this error when running node 4.4.5

```
/node_modules/async-hook/async-hook.js:106
  asyncWrap.setupHooks({
            ^
TypeError: init callback must be a function
    at TypeError (native)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

**Comment by AndreasMadsen** [comment]
Known issue. This is fixed in node v4.5 (not yet released). A work around is to replace `async-hook` with version 1.2.0.

Follow the v4.5 release here https://github.com/nodejs/node/pull/7688

**Comment by AndreasMadsen** [comment]
v4.5.0 is now available.
